Bottom: f558a7903be244d8a2a0de3856d3328c7834508a
Top:    e7a9a198982f62970fdf43ffd0c25b449b07c3e3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 03:06:56 +0100

Refresh of use-trackball-to-allow-rotatio

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 92c2def..c7968a0 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -935,8 +935,7 @@ ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
                              -gport->view_x0,
                 ghid_flip_y ? gport->view_y0 - PCB->MaxHeight :
                              -gport->view_y0, 0);
-//  hid_expose_callback (&ghid_hid, &region, 0);
-  hid_expose_callback (&ghid_hid, NULL, 0);
+  hid_expose_callback (&ghid_hid, global_view_2d ? &region : NULL, 0);
   hidgl_flush_triangles (&buffer);
   glPopMatrix ();
